sdks/java/io/hadoop-input-format/src/test/java/org/apache/beam/sdk/io/hadoop/inputformat/unit/tests/inputs/Employee.java

```diff
@@ -0,0 +1,74 @@
+package org.apache.beam.sdk.io.hadoop.inputformat.unit.tests.inputs;
+
+import org.apache.beam.sdk.coders.AvroCoder;
+import org.apache.beam.sdk.coders.DefaultCoder;
+import org.apache.beam.sdk.io.hadoop.inputformat.HadoopInputFormatIO;
+/**
+ * This class is Employee POJO class with properties- employee name and address. 
+ * Used in {@linkplain HadoopInputFormatIO} for different unit tests
+ */
+@DefaultCoder(AvroCoder.class)
+public class Employee {
```

Comment:
any reason not to use AutoValue? that should give you equals/hashcode implementations for free.